cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -176,4 +187,161 @@ describe('FluxQueryBuilder', () => {
       // numbers of API calls that are time consuming and unnecessary
     })
   })
+
+  describe('Schema Composition', () => {
+    beforeEach(() => {
+      window.sessionStorage.setItem(
+        'dataExplorer.schema',
+        JSON.parse(JSON.stringify(DEFAULT_SCHEMA))
+      )
+      window.sessionStorage.setItem('dataExplorer.query', '')
+
+      cy.setFeatureFlags({
+        schemaComposition: true,
+      }).then(() => {
+        cy.wait(1200).then(() => {
+          cy.reload()
+          cy.getByTestID('flux-sync--toggle')
+        })
+      })
+    })
+
+    const bucketName = 'defbuck'
+    const measurement = 'ndbc'
+
+    const selectSchema = () => {
+      cy.log('select bucket')
+      cy.getByTestID('bucket-selector--dropdown-button').click()
+      cy.getByTestID('bucket-selector--search-bar').type(bucketName)
+      cy.getByTestID(`bucket-selector--dropdown--${bucketName}`).click()
+      cy.getByTestID('bucket-selector--dropdown-button').should(
+        'contain',
+        bucketName
+      )
+
+      cy.log('select measurement')
+      cy.getByTestID('measurement-selector--dropdown-button')
+        .should('be.visible')
+        .should('contain', 'Select measurement')
+        .click()
+      cy.getByTestID('measurement-selector--dropdown--menu').type(measurement)
+      cy.getByTestID(`searchable-dropdown--item ${measurement}`)
+        .should('be.visible')
+        .click()
+      cy.getByTestID('measurement-selector--dropdown-button').should(
+        'contain',
+        measurement
+      )
+    }
+
+    const confirmSchemaComposition = () => {
+      cy.getByTestID('flux-editor', {timeout: 30000})
+        // we set a manual delay on page load, for composition initialization
+        .contains(`from(bucket: "${bucketName}")`, {timeout: 30000})
+      cy.getByTestID('flux-editor').contains(
+        `|> filter(fn: (r) => r._measurement == "${measurement}")`
+      )
+      cy.getByTestID('flux-editor').contains(
+        `|> yield(name: "_editor_composition")`
+      )
+    }
+
+    describe('basic ability:', () => {
+      it('can make a composition block', () => {
+        cy.getByTestID('flux-editor', {timeout: 30000})
+
+        cy.log('modify schema browser')
+        selectSchema()
+
+        cy.log('editor text contains the composition')
+        confirmSchemaComposition()
+      })
+    })
```

Comment:
Since line 275 and 276 already cover the same test scenario, is it possible to remove the 'basic ability' test block to speed up ci tests?